ibis/backends/tests/conftest.py

```diff
@@ -64,7 +64,20 @@ def _get_backends_to_test():
     if not backends_raw:
         return []
 
-    backends = backends_raw.split()
+    if backends_raw == "all":
+        backends = set(_get_all_backends())
+        # spark is just an alias to pyspark so don't re-run all spark tests
+        backends.discard("spark")
+    else:
+        backends = set(backends_raw.split())
+
+    if filter_out is not None:
+        backends = backends.difference(filter_out.split(" "))
```

Comment:
I think you can use `.split()` with no arguments, and it'll split on one or more whitespace characters.